react/kanban/src/components/Board/Column/Column.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ColumnSectionCards from './ColumnSectionCards';
+
+function Column({ column, addCard }) {
+  const { id, title, color } = column;
+  
+  const handleAddCard = () => {
+    const title = prompt('Card title:', '');
```

Comment:
try to replace prompt with inline editor mechanism, you already have it for editing